debian_packages.yaml

```diff
@@ -42,7 +42,7 @@
 
 # python and java only build on a subset of architectures
 - distros: ["debian11"]
-  archs: ["amd64","arm64"]
```

Comment:
I think I would just separate out the python and java deps here. It's pretty expensive to download some of these deps, and if python isn't building on s390x we shouldn't be fetching them (we prefetch all deps in CI)

Reply:
Sure, I can do that.

Reply:
Done with latest commit.